src/main/java/lt/andriaus/hangman/usecase/BasicGameManager.java

```diff
@@ -0,0 +1,34 @@
+package lt.andriaus.hangman.usecase;
+
+import lt.andriaus.hangman.database.Database;
+import lt.andriaus.hangman.domain.Game;
+
+import java.util.*;
+
+public class BasicGameManager implements GameManager {
+    private final Database<String> wordDB;
+    private final Database<Game> gameDB;
+
+    public BasicGameManager(Database<String> wordDB, Database<Game> gameDB) {
+        this.wordDB = wordDB;
+        this.gameDB = gameDB;
+    }
+
+    public int createGame() {
+        Optional<String> randomWord = wordDB.loadRandom();
+        if (randomWord.isEmpty())
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.